modules/tinymce/src/core/main/ts/newline/NewLineUtils.ts

```diff
@@ -224,6 +227,47 @@ const createNewBlock = (
   return block;
 };
 
+const flattenFontSize = (block: SugarElement, node: SugarElement, fontSize: string | null = null) => {
+  if (block.dom === node.dom) {
```

Comment:
If you're going to use Sugar, use Sugar 😉 
```suggestion
  if (Compare.eq(block, node)) {
```

- [ ] And the other two versions of this below